onchain/rollups/test/foundry/util/TestBase.sol

```diff
@@ -17,4 +19,16 @@ contract TestBase is Test {
         vm.assume(addr != MULTICALL3_ADDRESS);
         _;
     }
+
+    function encodeEvmAdvance(
```

Comment:
I don't think this is the best place for this function.
You can create a separate helper library under `test/foundry/util` just for this task, if you want. But this contract is inherited by many test contracts, many of which have nothing to do with inputs.